src/promptflow-tracing/promptflow/tracing/_integrations/_openai_injector.py

```diff
@@ -70,7 +70,9 @@ def inject_headers(kwargs):
         injected_headers = get_aoai_telemetry_headers()
         original_headers = kwargs.get("headers" if IS_LEGACY_OPENAI else "extra_headers")
         if original_headers and isinstance(original_headers, dict):
-            injected_headers.update(original_headers)
+            injected_headers.update(
```

Comment:
Thanks @weedqian for the suggestion. My aim is to use the more efficient code here since this is called for each call made to LLM. 

The dictionary comprehension is more efficient in terms of both readability and performance. It is recommended to use the dictionary comprehension approach for this task.

Reply:
I don't think there are any big differences on the perf. 

We are an open source project. The readability of the code is critical.

Reply:
Done thanks.